test/utils/deploy/deploy.go

```diff
@@ -71,7 +71,10 @@ func NewForPlatform(plat platform.Platform) Deployer {
 }
 
 func GetDeploymentWithSched(ctx context.Context) (NroDeploymentWithSched, error) {
-	sd := NroDeploymentWithSched{}
+	sd := NroDeploymentWithSched{
+		Deployer: NewForPlatform(configuration.Plat),
```

Comment:
this seems correct

Reply:
we need to set it first thing in case the cluster didn't have the scheduler and fails to get it